src/HttpWorker.php

```diff
@@ -56,13 +62,25 @@ public function waitRequest(): ?Request
             return null;
         }
 
+        if (static::$codec === null) {
+            static::$codec = json_validate($payload->header) ? Frame::CODEC_JSON : Frame::CODEC_PROTO;
```

Comment:
```suggestion
            static::$codec = \json_validate($payload->header) ? Frame::CODEC_JSON : Frame::CODEC_PROTO;
```

Can we avoid symfony polyfill here?